static/docs/use-cases/data-storage-on-external-drive.md

```diff
@@ -0,0 +1,207 @@
+# Data Storage On External Hard Drive
+
+Sometimes the data may be stored on an
+[external hard drive](https://whatis.techtarget.com/definition/external-hard-drive).
+Usually such data are huge, which means that they won't fit on our home
+directory, and even if they did, it would certainly take a long time to copy
+them back and forth from the external drive to the internal one. For example
+let's say that the size of the external drive is 16TB, while the hard drive of
+our home directory is only 320GB.
+
+In this case we would like to process the data where they are located (on the
+external drive). We also would like to save the results there, and certainly to
+store the <abbr>cached</abbr> files there as well.
+
+The most easy way to do this would be to locate the <abbr>workspace</abbr> on
+the external drive itself. If we assume that the external drive is mounted on
+`/mnt/data/`, then it could be done like this:
+
+```dvc
+$ sudo su
+# cd /mnt/data/
+# git init
+# dvc init
+```
+
+But in case this is not possible (or is not preferable), we can easily setup the
+workspace in our home directory, while all the data files and their caches keep
+staying on the external drive. DVC will still be able to track them properly.
+
+## Make the data directory accessible
+
+For this to work, first you have to make sure that you can read and write the
+data directory `/mnt/data/`. The most straightforward way to do this is by
+setting proper ownership and permissions to it, like this:
+
+```dvc
+$ sudo chown <username>: -R /mnt/data/
+$ chmod u+rw -R /mnt/data/
+```
+
+## Start a DVC project and setup an external cache
+
+An _external_ <abbr>cache</abbr> is called so because it resides outside of the
+workspace directory. Let's create a directory for it on `/mnt/data/`:
```

Comment:
Hmmm. Typically we mean online by "external" in _external cache_. See https://dvc.org/doc/commands-reference/config#cache.

And instead of `_external_ <abbr>cache</abbr>` here I would suggest a link: `[external cache](/doc/user-guide/external-outputs)`.

Reply:
I think I was wrong about the meaning of _external_ but I stand by the link suggestion.

Reply:
Linking.